### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by grongor**
#crymore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ I want to list those WTFs so it either reminds me to get understanding of it or
 
 ## Missing access modifiers
 
-`type memberInfo struct {` means public access
+`type memberInfo struct {` means private access (only accessible within single package)
 
-`type MemberInfo struct {` means private access
+`type MemberInfo struct {` means public access (accessible fron other packages)
```
Comment (simPod) [inline README.md]: typo

**Review by simPod — APPROVED** [review]
Overall, lookin' good!

maybe we could add `#crymore` to the readme as well?

**Comment by grongor** [comment]
sure, but that's outside the scope of this pr :joy:

**Comment by simPod** [comment]
Kk will add it to backlog. Thx! 

_Hope you're subscribed to Hacktober fest!_ 😂 

:shipit:

**Pull request merged**